recipes/piranha/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "0.5" %}
+
+package:
+  name: piranha
+  version: {{ version }}
+
+source:
+  fn: piranha-{{ version }}.tar.gz
+  url: https://github.com/bluescarni/piranha/archive/v{{ version }}.tar.gz
+  sha256: 34a89bda8208ff48cfb116efa7d53c09e8a9b3838af4bb96ba2e19e4930b3a58
+
+build:
+  number: 0
+  skip: true        # [win]
+
+requirements:
+  build:
+    - toolchain
+    - cmake
+    - gmp
+    - mpfr
+    - boost 1.62.*
+    - bzip2 1.0.*
```

Comment:
Given this is header only, do we actually need any of these (excepting `cmake`)?